cli/azd/pkg/devcenter/environment_store.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	ConfigPath                                 = "devCenter"
+	ConfigPath                                 = "platform.config"
```

Comment:
Is there anything we can do in product now to make this more extensible? Like a package of "devcenter" things that fall into a "platform" provider type object?  I'm concerned about there being devcenter littered throughout code and knowing that that we'll have to refactor later.  Just thinking if there's anything we can do now